packages/core/src/profiling.ts

```diff
@@ -0,0 +1,49 @@
+import { logger } from '@sentry/utils';
+import { ProfilingIntegration, Profiler } from '@sentry/types';
+
+import { DEBUG_BUILD } from "./debug-build";
+import { getClient } from "./currentScopes";
+
+/**
```

Comment:
These two calls are just thin wrappers around getIntegration and calling a method on that integration.